keyboards/hotdox/keymaps/via/rules.mk

```diff
@@ -1,12 +1,13 @@
 VIA_ENABLE = yes
 LTO_ENABLE = yes
 
-MOUSEKEY_ENABLE         = no  # Mouse keys
-EXTRAKEY_ENABLE         = no # Audio control and System control
+MOUSEKEY_ENABLE         = yes # Mouse keys
+EXTRAKEY_ENABLE         = yes # Audio control and System control
 
-LEADER_ENABLE			= no
-FAUXCLICKY_ENABLE		= no
-MIDI_ENABLE				= no
-BLUETOOTH_ENABLE		= no
-KEY_LOCK_ENABLE			= no
-TERMINAL_ENABLE			= no
\ No newline at end of file
+
+LEADER_ENABLE			      = no
+FAUXCLICKY_ENABLE		    = no
+MIDI_ENABLE			      	= no
+BLUETOOTH_ENABLE    		= no
+KEY_LOCK_ENABLE		    	= no
+TERMINAL_ENABLE		    	= no
```

Comment:
```suggestion
```
Doesn't look like any of these are needed as they're all `no` by default and not mentioned in the keyboard-level rules.mk.